### ISSUE TITLE
detect sanitization in else if below

### ISSUE THREAD

**Issue by fbm-static** [WordPress-Coding-Standards/WordPress-Coding-Standards#411]
```php
if ( false ) {

} elseif ( isset( $_GET['id'] ) && isset( $_GET['uniqid'] ) ) { // Detected access of super global $_GET...

	$job = self::get( intval( $_GET['id'] ) ); //input var okay
	$uniqid = sanitize_text_field( $_GET['uniqid'] ); //input var okay
}
```

I'm sorry I don't have a better wording for the title.

**Comment by JDGrimes** [comment]
This isn't related to sanitization. The error is from the `WordPress.VIP.SuperGlobalInputUsage` sniff, not `WordPress.VIP.ValidatedSantizedInput`. I don't use the `SuperGlobalInputUsage` sniff, but I believe its job is to flag all uses of the input superglobals (`$_GET`, `$_POST`, etc.) without any regard to how they are being used. So I think that this is expected behavior.
